<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chartcreate</class>
 <widget class="QDialog" name="chartcreate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>209</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>utiliser des pourcentage</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="chart_preview" native="true">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>331</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stack_form_chart">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>124</y>
     <width>351</width>
     <height>81</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>99</x>
       <y>45</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Faire une moyenne</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>0</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>26</x>
       <y>3</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Population ciblé :</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QComboBox" name="comboBox_response">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>3</y>
       <width>181</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_5">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>43</y>
       <width>181</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>6</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Réponse donné :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>47</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Catégories  :</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>35</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Question :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_question">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>310</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>0</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Apercu:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Réponse à considérer :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="type_chart_combo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Toutes les réponses</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>réponse 1</string>
    </property>
   </item>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
